sbx/trusted_access_nodes_list.yaml

```diff
@@ -7,3 +7,7 @@ organizations:
     publicKey: 0x0440051e727c96f5dae8900648327c058f266d9ea58758a624781b0ae0453c1b3a90e1690c1ae30dcf7e84c97fd2061267e70634211425acfc0e57c4363e4fafe8
     url: https://desmos.dome-marketplace-lcl.org
     dlt_address: 0xea375D05cfb3A723208C3Cce3B843D0Dd1117086
+  - name: ENG
+    publicKey: 0x04e09816286522738db42a9dba347214513f5b49de91001e9c029246bd612aadbff12596492ee2d656d7e8ef9e47f17b82facb31726904046cb8df847345dc0de3
+    url: https://dome-dev.eng.it
```

Comment:
Is this URL pointing to the Replication Service (Desmos) component?

Reply:
Hi Oriol, regarding the URL from the doc: 
"The url must be the URL of the organization's Access Node, and it can be an IP address or a domain."
I don't know what I have to set, please let me know!
Thanks a lot

Reply:
This URL is which you will configure as a URL of the desmos (Replication Service) component. The only one who is exposed to Internet.

Reply:
Hi Oriol, you mean the externalDomain. I'm using `externalDomain: https://dome-dev.eng.it` in the values.yaml